roundware/api2/filters.py

```diff
@@ -242,10 +242,14 @@ class Meta:
 class SpeakerFilterSet(django_filters.FilterSet):
     activeyn = django_filters.TypedChoiceFilter(choices=BOOLEAN_CHOICES, coerce=strtobool)
     project_id = django_filters.NumberFilter()
+    parent_ids_or = IntegerListFilter(field_name='parents', lookup_expr='in') # performs OR filtering
+    parent_ids = IntegerListAndFilter(field_name='parents__id') # performs AND filtering
+    children_ids_or = IntegerListFilter(field_name='children', lookup_expr='in') # performs OR filtering
```

Comment:
do you use the `_or` naming convention elsewhere in the application? 

to me it might be a bit more clear as

parent_ids_include

or something. nitpick :)

Reply:
I do actually use this same nomenclature for filtering assets by tags, so I was trying to be consistent. Not convinced it it the best convention, but probably best to stick with it for consistency?

`tag_ids_or = IntegerListFilter(field_name='tags', lookup_expr='in')`

Reply:
Agree, consistency is good!